WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -536,7 +536,12 @@ public static void signOut(Context context) {
             AppLog.e(T.UTILS, "Error while cleaning the Local KeyStore File", e);
         }
 
-        wpDB.deleteAllAccounts();
+        // Set a shared preference to signify that this user signed out
+        SharedPreferences settings = PreferenceManager.getDefaultSharedPreferences(context);
+        SharedPreferences.Editor editor = settings.edit();
+        editor.putBoolean(IS_SIGNED_OUT_PREFERENCE, true);
+        editor.commit();
```

Comment:
This is a minor point: Google recommends using `apply` rather than `commit` - [here's why](http://developer.android.com/reference/android/content/SharedPreferences.Editor.html#apply()).